mathesar_ui/src/pages/schema/SchemaPage.svelte

```diff
@@ -81,6 +83,13 @@
   }
 
   $: isDefault = schema.name === 'public';
+  onMount(async () => {
+    logEvent('opened_schema', {
+      database_name: database.name,
+      schema_name: schema.name,
+      page: 'schema',
```

Comment:
What does `page` refer to here? It's a little vague so I'd prefer a more verbose name.

Reply:
@kgodey `page` refers to the screen/page where the event/action took place. Does the name `event_location`  or `source_page`  convey the intent better?


In case you want to understand the reason behind the `page` attribute. It is useful in case an action can be performed in more than one place. For example, we can create a Table from
- The schema overview page (/<db_name>/<schema_id>/) . The event will be `opened_create_table` and the associated metadata will be `page: schema`.  
- The tables page  (/<db_name>/<schema_id>/tables/). The event will be `opened_create_table` and the associated metadata will be `page: tables`.  
The `page` attribute is helpful if we want to analyse the preferred page to create a Table.

Reply:
Makes sense @silentninja. I would do something like "source": "schema_page" since not all starting points are pages (e.g. the record selector might be a source).